control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -428,45 +430,47 @@ func (r *reconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Result
 		}
 	}
 
-	log.Info("reconciling oauth serving cert ca bundle")
-	if err := r.reconcileOAuthServingCertCABundle(ctx, hcp); err != nil {
-		errs = append(errs, fmt.Errorf("failed to reconcile oauth serving cert CA bundle: %w", err))
-	}
-
 	log.Info("reconciling user cert CA bundle")
 	if err := r.reconcileUserCertCABundle(ctx, hcp); err != nil {
 		errs = append(errs, fmt.Errorf("failed to reconcile user cert CA bundle: %w", err))
 	}
 
-	log.Info("reconciling oauth browser client")
-	oauthBrowserClient := manifests.OAuthServerBrowserClient()
-	if _, err := r.CreateOrUpdate(ctx, r.client, oauthBrowserClient, func() error {
-		return oauth.ReconcileBrowserClient(oauthBrowserClient, r.oauthAddress, r.oauthPort)
-	}); err != nil {
-		errs = append(errs, fmt.Errorf("failed to reconcile oauth browser client: %w", err))
+	log.Info("reconciling oauth serving cert ca bundle")
+	if err := r.reconcileOAuthServingCertCABundle(ctx, hcp); err != nil {
+		errs = append(errs, fmt.Errorf("failed to reconcile oauth serving cert CA bundle: %w", err))
```

Comment:
Is there a reason for moving the `oauth serving cert ca bundle` reconcile to after the `user cert CA bundle` ?

Reply:
ha, no.  at first, i moved it inside the `if` block below, but then realized we still need to reconcile it.